src/toastAndroid.rei

```diff
@@ -0,0 +1,17 @@
+type duration =
+  | BOTTOM
+  | CENTER
```

Comment:
is duration `CENTER` really a thing :D ?

Reply:
Fixed :D